components/automate-ui/src/app/pages/data-feed/data-feed.component.ts

```diff
@@ -160,40 +160,67 @@ export class DataFeedComponent implements OnInit, OnDestroy {
     this.deleteModalVisible = false;
   }
 
+  public addHeadersforCustomDataFeed(customHeaders: string): {} {
+    //  const customHeaders: string = header;
+        const headersJson = {};
+        const headersVal = customHeaders.split('\n');
+        for (const values in headersVal) {
+          const word = headersVal[values].split(':');
+          headersJson[word[0]] = word[1];
+        }
```

Comment:
```suggestion
        for (const values of headersVal) {
          const word = values.split(':');
          headersJson[word[0]] = word[1];
        }
```
For iterating an array please use 'for of' loop.